osmosis-1/osmosis-1.assetlist.json

```diff
@@ -10910,6 +10910,46 @@
         "osmosis-main",
         "osmosis-frontier"
       ]
+    },
+    {
+        "description": "$DGL is a utility token for the Licorice ecosystem and a native token on Juno chain.",
+        "denom_units": [
+          {
+            "denom": "ibc/D69F6D787EC649F4E998161A9F0646F4C2DCC64748A2AB982F14CAFBA7CC0EC9",
+            "exponent": 0,
+            "aliases": [
+              "factory/juno1u805lv20qc6jy7c3ttre7nct6uyl20pfky5r7e/DGL"
+            ]
+          },
+          {
+            "denom": "dgl",
+            "exponent": 6
+          }
+        ],
+        "type_asset": "ics20",
+        "address": "juno1u805lv20qc6jy7c3ttre7nct6uyl20pfky5r7e",
+        "base": "ibc/D69F6D787EC649F4E998161A9F0646F4C2DCC64748A2AB982F14CAFBA7CC0EC9",
+        "name": "DGL Licorice",
+        "display": "dgl",
+        "symbol": "DGL",
+        "traces": [
+          {
+            "type": "ibc",
+            "counterparty": {
+              "chain_name": "juno",
+              "base_denom": "factory/juno1u805lv20qc6jy7c3ttre7nct6uyl20pfky5r7e/DGL",
+              "channel_id": "channel-0"
+            },
+            "chain": {
+              "port": "transfer",
```

Comment:
Checks are failing because port is not allowed here. Also, this entire page is not required as this file is entirely generated

Reply:
Would you like me to remove the entire file?

Reply:
Might as well, yes.

Reply:
> Might as well remove your changes, yes.

Done, assetlist.json file removed.  Couple questions.  Can you return the extra 900 OSMO that was taken from my account by error, so that I can add more liquidity to pool #1143 please.  Also, what needs to be done on our end to have the asset verified, and on the main page of Osmosis.Zone ?

Thank you

Reply:
For listing as a verified asset: We continue to work on refining the listing requirements, which can be found [here](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md), but if neither not built on Osmosis nor approved by Osmosis governance, it would require evidence of legitimacy, including a decent CoinGecko Market Cap rank and significant following and brand presence.

Reply:
Regarding the pool creation fee, the latest chain was upgrade changed the pool creation fee to 1000 OSMO. The difference between the advertised fee and the amount taken from your wallet will be investigated further by the team (we understand that the frontend indeed only showed 100 OSMO creation fee at the time). You should be contacted shortly about this.

Reply:
> Regarding the pool creation fee, the latest chain was upgrade changed the pool creation fee to 1000 OSMO. The difference between the advertised fee and the amount taken from your wallet will be investigated further by the team (we understand that the frontend indeed only showed 100 OSMO creation fee at the time). You should be contacted shortly about this.   

Thanks for the explanation.  Was the fee increase an error made in the upgrade or was it intended?  There's a pretty dramatic increase there.

Reply:
I think it was unintentional